portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase

```diff
@@ -490,4 +490,37 @@ definition {
 		}
 	}
 
+	@description = "Add a new admin user, the user creates a new page and web content, and adds a web content widget to the page"
```

Comment:
Don't send another commit to fix your original one, you should update within your previous commit.